hapi-loader/test_producer/requirements.txt

```diff
@@ -0,0 +1 @@
+confluent_kafka
```

Comment:
Did you mean to push this folder?

Reply:
yes, it's useful for testing and I explained how to use it in the README